examples/dtls-echo/dtls-client.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015 Freie Universität Berlin
+ * Copyright (C) 2018 Inria
```

Comment:
You should keep the original copyright from FU Berlin and add the Inria one below

Reply:
Thanks! I was not sure how to handle this.